### PR TITLE
#Task3 Changed apiPaths

### PR DESCRIPTION
**Pull request by Partyshe**
Added new path to Products.

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@
 #
 # Happy Coding!
 
-service: partyshe-first-app
+service: partyshe-third-app-service
```
Comment (SergeyKovalchuk) [inline serverless.yml:14]: This MUST be the same service. 

You have ONE application

Comment (SergeyKovalchuk) [reply serverless.yml:14]: Would you like to create new application per feature?

File: serverless.yml [hunk 2]
```diff
@@ -23,14 +23,16 @@ frameworkVersion: '2'
...
+  region: eu-west-1
 
 plugins:
   - serverless-finch
   - serverless-single-page-app-plugin
 
 custom:
   client:
-    bucketName: partyshe-second-app
+    bucketName: partyshe-third-app
```
Comment (SergeyKovalchuk) [inline serverless.yml:35]: Bucket should be the same

Comment (Partyshe) [reply serverless.yml:35]: Thank you, Sergey. I understand. I've deleted all services and buckets and played with deployment. 
If naming is not critical and if you don't mind, I want to keep current version.

File: serverless.yml (hunk 1 above)
Comment (Partyshe) [reply serverless.yml:14]: Thank you, Sergey. I understand. I've deleted all services and buckets and played with deployment. 
If naming is not critical and if you don't mind, I want to keep current version.

**Pull request merged**